spec/index/index_spec.rb

```diff
@@ -280,4 +280,11 @@
       it { expect { idx.at 2, 3 }.to raise_error IndexError }
     end
   end
-end
\ No newline at end of file
+
+  context "Enumerable" do
```

Comment:
Hmm yes that works. In that case, can you write an `it` block inside the `context 'Enumerable'` which specifies that the test is for all Enumerable methods? That way future contributors won't be confused.